.gitignore

```diff
@@ -0,0 +1,19 @@
+.idea
+*.iml
```

Comment:
I personally keep these in a global ignore list on my systems. It's not bad you just have to imagine how the file would look like if we would add all kinds of project files (vscode, sublime, eclipse, ...)

Reply:
Alright, removed :)